openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -60,32 +60,35 @@ This specification uses the terms "Access Token", "Authorization Endpoint", "Aut
 
 This specification also defines the following terms. In the case where a term has a definition that differs, the definition below is authoritative for this specification.
 
-Credential:
-:  A set of one or more claims about a subject made by a Credential Issuer. Note that this definition of a term "credential" in this specification is different from that in [@!OpenID.Core] and [@!RFC6749].
+Credential Dataset:
+:  A set of one or more claims about a subject made by a Credential Issuer.
 
-Verifiable Credential (VC):
-:  An Issuer-signed Credential whose integrity can be cryptographically verified. It can be of any format used in the Issuer-Holder-Verifier Model, including, but not limited to those defined in [@VC_DATA] and [@ISO.18013-5].
+Verifiable Credential (or Credential):
+:  An Issuer-signed Credential Dataset whose integrity can be cryptographically verified. It can be of any Credential Format.
```

Comment:
I'd like to keep "Credential" in the first line to make it easier to find.

Reply:
Using my mental compiler, the alias comes after the definitions .. but not relevant if you feel more comfortable an editorial pattern instead of another (we're not machines!)

Reply:
I agree with @danielfett, its good for it to appear as an alias in the definition name rather than buried in the definition. We could opt to have both e.g add the last sentence in @peppelinux suggestion to this definition?

Reply:
change updated